$NetBSD: patch-ah,v 1.3 1999/07/08 12:48:42 agc Exp $

Solaris redefines rlimit to rlimit64, to catch the rlimit() call when
function names are re-defined.  Unfortunately, it also does that after
defining "struct rlimit", and so we lose because it can't find the
structure.  So we get to redefine it ourselves before <sys/resource.h>
is included.  Thanks, guys.

--- sh.h	1998/09/18 16:09:13	3.85
+++ sh.h	1999/03/08 13:14:06
@@ -224,6 +224,7 @@
 #   include <sys/sysinfo.h>
 #  else
 #   define sysrusage rusage
+#   define rlimit rlimit64
 #   include <sys/resource.h>
 #  endif /* convex */
 # endif /* SYSVREL>3 */
